main.go

```diff
@@ -190,7 +190,7 @@ func main() {
 		MetricsBindAddress: metricsAddr,
 		Port:               9443,
 		LeaderElection:     enableLeaderElection,
-		LeaderElectionID:   "d73494fd.storageos.com",
+		LeaderElectionID:   "api-manager-lease",
```

Comment:
With this name change, we are losing the information that this belongs to storageos. Since our current default deployment is in kube-system namespace, along with many other system resources, it'll make it harder to identify that this belongs to storageos, if we want to find that out for some reason.
Maybe `api-manager-lease.storageos.com` would be better to keep the ownership info in the name?
Just an opinion.

Reply:
Good point, but my preference would be to use the `storageos-` prefix we use for other resources, so something like `storageos-api-manager-lease`.